### ISSUE TITLE
fin: Enable BatchNorm in fin - Applicability

### ISSUE THREAD

**Issue by alexandraBara** [ROCmSoftwarePlatform/MITunaX#646]
Adding applicability to Fin for batch norm

Reviewer: @technicalgrp89
